### ISSUE TITLE
[Screen Reader - Open and close New Tab]: Screen reader does not provide any confirmation after opening or closing new tab.

### ISSUE THREAD

**Issue by Chandresh1007** [microsoft/terminal#8464]
**User Experience:**
Users who rely on screen reader would be impacted and not able to get the information as narrator stays silent and does not provide any confirmation about the tab getting open or closed.

**Test Environment:**
OS: Win 10 2004 Build (20262.1010)
App: Windows Terminal
App version: 1.4.3243.0

**Repro Steps:**
1. Open Windows terminal app.
2. Launch Narrator.
3. Press "Ctrl+Shift+T" or "Ctrl+Shift+W" to open or close tab respectively.
4. Observe the issue.

**Actual Result:**
Screen reader does not provide any confirmation and stays silent after opening or closing new tab.

**Expected Result:**
Screen reader should provide the proper confirmation about the opening or closing new tab.
When pressing "Ctrl+Shift+T" to **open new tab narrator should announce** the information like **new tab opened successfully**, Similarly pressing "Ctrl+Shift+W" to **close tab, narrator should announce** the information like **tab closed successfully**.

**Note:** Issue does repro with NVDA. 

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7Bb19c05bd-d82f-4d6d-a8e5-7081bf5c2dba%7D&cid=eea52d2d-e8b6-489a-95ad-9e5bfc7720c7

[MAS4.2.12_opening or closing tab_no confirmation provided by narrator.zip](https://github.com/microsoft/terminal/files/5629845/MAS4.2.12_opening.or.closing.tab_no.confirmation.provided.by.narrator.zip)

**Comment by zadjii-msft** [comment]
@carlos-zamora Does chromium do this? Do we need to do this for panes too?

**Comment by carlos-zamora** [comment]
> Does chromium do this?

Just tested it. It does.

> Do we need to do this for panes too?

So, chromium edge lets you open a pane with dev tools using F12. Surprisingly, that _does not_ notify the user that a new pane has opened/closed. It just starts following your keyboard focus.

That said, I'm thinking we actually _should_ notify the user when a new pane is opened/closed. In fact, we should take it a step further, and provide a notification any time the layout changes (i.e. open/close/toggle-zoom pane). Maybe even whenever the user invokes a keybinding? Screen readers generally listen to these kinds of events and do work on their end to update their understanding of what's going on. That's how generated output works.

After some investigation, the TabView in the XAML Controls Gallery seems to have this issue as well. In both, we properly notify when we switch tabs so at least we have that going for us haha.

**Comment by Chandresh1007** [comment]
WinUI 2 team has accepted this as a XAML bug.
TabView: Screen reader does not provide any confirmation after opening/closing new tab · Issue #3746 · microsoft/microsoft-ui-xaml (github.com)

Hence, closing the bug as external.
